_config.yml

```diff
@@ -26,8 +28,8 @@ permalink:        pretty
 
 # Setup
 title:            The OpenTracing Project
-tagline:          'Common APIs for distributed tracing and context propagation'
-description:      'Common APIs for distributed tracing and context propagation'
+tagline:          'Consistent and expressive APIs for distributed tracing and context propagation'
+description:      'Consistent and expressive APIs for distributed tracing and context propagation'
```

Comment:
yeah, that fix is pending in my next round of updates... will send that out later today for sure. (yesterday was lost to non-laptop work)